pydov/search/interpretaties.py

```diff
@@ -242,3 +243,125 @@ def search(self, location=None, query=None, return_fields=None):
+                LithologischeBeschrijvingenSearch.__wfs_schema,
+                LithologischeBeschrijvingenSearch.__fc_featurecatalogue)
+
+            for field in fields.values():
+                if field['name'] not in self._type.get_field_names(
+                        include_wfs_injected=True):
+                    self._type._fields.append({
+                        'name': field['name'],
+                        'source': 'wfs',
+                        'sourcefield': field['name'],
+                        'type': field['type'],
+                        'wfs_injected': True
+                    })
+
+            self._fields = self._build_fields(
+                LithologischeBeschrijvingenSearch.__wfs_schema,
+                LithologischeBeschrijvingenSearch.__fc_featurecatalogue)
+
+    def search(self, location=None, query=None, return_fields=None):
+        """Search for 'lithologische beschrijvingen'. Provide either
+        `location` or `query`. When `return_fields` is None, all fields
+        are returned.
+
+        Parameters
+        ----------
+        location : tuple<minx,miny,maxx,maxy>
+            The bounding box limiting the features to retrieve.
+        query : owslib.fes.OgcExpression
+            OGC filter expression to use for searching. This can contain any
+            combination of filter elements defined in owslib.fes. The query
+            should use the fields provided in `get_fields()`. Note that not
+            all fields are currently supported as a search parameter.
+        return_fields : list<str> or tuple<str> or set<str>
+            A list of fields to be returned in the output data. This should
+            be a subset of the fields provided in `get_fields()`. Note that
+            not all fields are currently supported as return fields.
+
+        Returns
+        -------
+        pandas.core.frame.DataFrame
+            DataFrame containing the output of the search query.
+
+        Raises
+        ------
+        pydov.util.errors.InvalidSearchParameterError
+            When not one of `location` or `query` is provided.
+
+        pydov.util.errors.InvalidFieldError
+            When at least one of the fields in `return_fields` is unknown.
+
+            When a field that is only accessible as return field is used as
+            a query parameter.
+
+            When a field that can only be used as a query parameter is used as
+            a return field.
+
+        pydov.util.errors.FeatureOverflowError
+            When the number of features to be returned is equal to the
+            maxFeatures limit of the WFS server.
+
+        AttributeError
+            When the argument supplied as return_fields is not a list,
+            tuple or set.
+
+        """
+        fts = self._search(location=location, query=query,
+                           return_fields=return_fields,
+                           extra_wfs_fields=['Type_proef', 'Proeffiche'])
```

Comment:
The `extra_wfs_fields` can be removed, since we assume all `Proeffiche` will be permanent keys of boreholes, we don't need the logic anymore to retrieve `Type_proef` to make the difference between `pkey_boring` and `pkey_sondering`. `Proeffiche` will then be a normal WFS field translated into `pkey_boring`.